scripts/cat/cats.py

```diff
@@ -42,31 +43,29 @@
     leader_ceremony_text_adjust,
 )
 
+class CatAgeEnum(Enum):
```

Comment:
```suggestion
class CatAgeEnum(StrEnum):
```
In order to work for earlier versions of python, this must be Strenum